qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -34,10 +35,7 @@
 from qiita_db.metadata_template.sample_template import SampleTemplate, Sample
 
 
-@qiita_test_checker()
-class TestPrepSample(TestCase):
-    """Tests the PrepSample class"""
-
+class SetUpTestPrepSample(TestCase):
```

Comment:
Very minor but it took me a little while to realize that SetUpTestPrepSample was a base class for TestPrepSample, maybe it would be better to change the name to be BaseTestPrepSample?